tests/functional/database-management.test.js

```diff
@@ -9,13 +9,107 @@ describe('database management', function () {
 
     context('database creation and deletion', function () {
 
+        (harnessConfigs.testRemoteSwarm ? context.skip : context)('permissioning', function () {
+
+            const randomUuid = `${Math.random()}`;
+
+            const notMasterKeyPair = {
```

Comment:
Best to use something to generate this key pair randomly. Also of note is that we're going to want to change the curve used with this key pair, so the more central that's specified, the better.

Reply:
Done